### DISCUSSION TITLE
About theme style

### DISCUSSION THREAD

**Discussion by Quinlivanner** [primefaces/community#2450] [PrimeVue Templates]
![image](https://github.com/user-attachments/assets/c48ab5be-a15e-467b-83a7-52f1e22213e1)

I want to use this style from primevue official website in my project, but I tried for a long time and didn't find the right style, can anyone tell me what theme is this?

**Comment by Quinlivanner** [comment]
![image](https://github.com/user-attachments/assets/9208f102-a17f-4948-8078-c7ac0d0985af)

here is my theme now.

**Comment by j0rgedev** [comment]
Hello @Quinlivanner. The default preset in Prime Vue Website is Aura, so you'll need to set this preset in your main.js file. The code shared is from [PrimeVue Configuration](https://primevue.org/theming/styled/#configuration)

```javascript
import PrimeVue from 'primevue/config';
import Aura from '@primevue/themes/aura';

const app = createApp(App);

app.use(PrimeVue, {
    // Default theme configuration
    theme: {
        preset: Aura,
        options: {
            prefix: 'p',
            darkModeSelector: 'system',
            cssLayer: false
        }
    }
 });
```
